src/modules/fw_att_control/FixedwingAttitudeControl.cpp

```diff
@@ -67,6 +67,8 @@ FixedwingAttitudeControl::FixedwingAttitudeControl(bool vtol) :
 	_yaw_ctrl.set_max_rate(radians(_param_fw_acro_z_max.get()));
 
 	_rate_ctrl_status_pub.advertise();
+	_spoiler_setpoint_with_slewrate.setSlewRate(1.f); //min 1s from no spoiler deflection to full deflection
+	_flaps_setpoint_with_slewrate.setSlewRate(1.f);
```

Comment:
can we make these "1.f" named constexpr?

Reply:
https://github.com/PX4/PX4-Autopilot/pull/19329/commits/98ea2aeb1c80f716c5335c194fbe02116df85ca6